sdk/core/core/inc/az_str.h

```diff
@@ -28,7 +29,18 @@
 // where
 //
 // `void some_function(az_const_span const span);`
-#define AZ_STR(STRING_LITERAL) (az_const_span) AZ_CONST_STR(STRING_LITERAL)
+#define AZ_STR(STRING_LITERAL) (az_span) AZ_CONST_STR(STRING_LITERAL)
+
+/**
+ * A callback that accepts zero-terminated string `char const *`.
+ */
+AZ_ACTION_TYPE(az_str_action, char const *)
+
+#define AZ_CR '\r'
```

Comment:
are you sure these should not all be "" literals?

Also: I encourage just writing them out again.

Reply:
RFC says `CR` so I use `CR`, not `\r` :-)